src/main/java/org/example/JDBCExample.java

```diff
@@ -0,0 +1,237 @@
+package org.example;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JDBCExample {
+
+    private static final String JDBC_DRIVER = "org.postgresql.Driver";
+    private static final String DB_URL = "jdbc:postgresql://localhost:5432/mydb";
+    private static final String USER = "postgres";
+    private static final String PASS = "root";
+
+    private static Connection connection = null;
+
+
+    static {
```

Comment:
avoid using static blocks especially for connecting to external resources. very nasty if they fail